### ISSUE TITLE
Universal sentence encoder unsupervised resume training

### ISSUE THREAD

**Issue by yoav-sparks** [tensorflow/hub#186]
Hi,
I would like to resume training of the USE in order to better fit my domain specific text (cryptocurrency related tweets) in order to cluster together tweets about the same subject.
I have looked at issues #46 #36 #110 but the thing is that I do not have any labels for the tweets.
Can I resume training with no labels?
Does it make sense to use random labels?
Do I have to try and collect labels even though it's a pure unsupervised problem?

Thanks,
Yoav

**Comment by r-wheeler** [comment]
Not tensorflow hub googler. You could use a loss function that computes the semantic similarity of the tweets as the optimization objective. [This model](https://github.com/eBay/Sequence-Semantic-Embedding) uses cosine similarity between a source and target and seems like it might fit your problem space.

**Comment by andresusanopinto** [comment]
This is generic ML question not related with hub. Closing.
